src/__tests__/e2e/integration.spec.ts

```diff
@@ -1,45 +1,49 @@
 import { PolywrapClient } from "@polywrap/client-js";
-import * as App from "../types/wrap";
-import path from "path";
 
-jest.setTimeout(60000);
+import { WebScraper } from "../types/wrap";
 
-describe("Web Scraper Wrapper End to End Tests", () => {
+jest.setTimeout(60000);
 
-  const client: PolywrapClient = new PolywrapClient();
-  let wrapperUri: string;
+describe("WebScraper", () => {
 
-  beforeAll(() => {
-    const dirname: string = path.resolve(__dirname);
-    const wrapperPath: string = path.join(dirname, "..", "..", "..");
-    wrapperUri = `fs/${wrapperPath}/build`;
-  })
+  const client = new PolywrapClient();
+  const localWrap = `file://${__dirname}/../../../build`
 
-  it("calls get_links", async () => {
-    const uri = "https://polywrap.io";
+  const webScraper = new WebScraper(client, undefined, localWrap);
 
-    const result = await client.invoke<App.Module_GetLinks>({
-      uri: wrapperUri,
-      method: "get_links",
-      args: { uri: uri }
+  it("get_links", async () => {
+    const result = await webScraper.get_links({
+      url: "https://webscraper.io/test-sites/e-commerce/allinone"
     });
-
     expect(result.ok).toBeTruthy();
     if (!result.ok) return;
-    // Add additional assertions as needed
+    expect(result.value).toContain(`/
+#page-top
+/
+/cloud-scraper
+/pricing
+#section3
+/documentation
+/tutorials
+/how-to-videos
+/test-sites
+https://forum.webscraper.io/
+https://chrome.google.com/webstore/detail/web-scraper/jnhgnonknehpejjnehehllkliplmbmhn?hl=en
+https://cloud.webscraper.io/
+/test-sites/e-commerce/allinone
+/test-sites/e-commerce/allinone/phones
+/test-sites/e-commerce/allinone/computers`
+    );
   });
 
-  it("calls get_text", async () => {
-    const url = "https://polywrap.io";
-
-    const result = await client.invoke<App.Module_GetText>({
-      uri: wrapperUri,
-      method: "get_text",
-      args: { url: url }
+  it("get_text", async () => {
+    const result = await webScraper.get_text({
+      url: "\nhttps://webscraper.io/test-sites/e-commerce/allinone\n"
```

Comment:
why does the url need newlines?

Reply:
It's just to make sure that the links are being "split" by `\n`. We can also add a feature where the user defines what the text delimiter is.

I think `\n` is a better default than ` `

Reply:
That change is great, but are the `\n` needed in the method argument?

Reply:
I think this might be a typo